neurons/validator.py

```diff
@@ -86,23 +89,38 @@ async def start(self, keypair: Keypair, port: int):
             # Add our custom routes
             self.register_routes()
 
-            # Initialize metagraph before starting server
             await self.sync_metagraph()
 
+            # TODO: fetch registered agents from API
+
             # Start background tasks
             asyncio.create_task(self.status_check_loop())
             asyncio.create_task(self.registration_check_loop())
             asyncio.create_task(self.check_registered_nodes_agents_loop())
 
             # Start the FastAPI server
-            config = uvicorn.Config(self.app, host="0.0.0.0", port=port, lifespan="on")
+            config = uvicorn.Config(
+                self.app, host="0.0.0.0", port=port, lifespan="on")
             server = uvicorn.Server(config)
             await server.serve()
 
         except Exception as e:
             logger.error(f"Failed to start validator: {str(e)}")
             raise
 
+    def create_scheduler(self):
+        logger.info("Generating queue...")
+        self.queue = generate_queue(self.registered_agents)
+        logger.info("Queue generated.")
+        self.scheduler = XSearchScheduler(
+            request_queue=self.queue,
+            interval_minutes=15,
+            batch_size=100,
+            priority=100,
+            search_count=10
```

Comment:
@hide-on-bush-x search count should be more than 10 tweets fetched @hide-on-bush-x :) Increase this significantly to ~450

Reply:
cc @grantdfoster